resources/translations/fi.edn

```diff
@@ -281,6 +281,9 @@
           :decision-requested {:subject "(%3) Päätöspyyntö"
                                :message "Hyvä %1,\n\n%2 on pyytänyt sinulta päätöstä käyttäjän %4 hakemuksesta %3.\n\nVoit tarkastella hakemusta osoitteessa %6"}
           :footer "\n\nTämä on automaattinen viesti. Älä vastaa."
+          :handler-reminder {:application "%1, lähettänyt %2"
+                             :message "Hyvä %1,\n\nSeuraavat hakemukset ovat vaiheessa ja voivat vaatia toimiasi:\n\n%2\n\nVoit tarkastella hakemuksia osoitteessa %3"
```

Comment:
This message could have a slightly more formal tone, e.g., "Seuraavien hakemusten käsittely on kesken, ja ne saattavat vaatia toimenpiteitä:"

Reply:
Similarly here, following Leena's proposal, it could be "Seuraavat hakemukset ovat avoinna ja saattavat vaatia toimenpiteitä:"